### PR TITLE
Add `Indonesia Convert Cup Open 2022` wiki article

### PR DESCRIPTION
**Pull request by Nivalyx**
requested by [Constantine](http://osu.ppy.sh/users/3221898) and [Urabe Mikoto](http://osu.ppy.sh/users/1272422) via pm

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Tournaments/de.md [hunk 1]
```diff
@@ -582,6 +582,7 @@ Nicht offizielle Turniere/Wettkämpfe, die von den Communities veranstaltet werd
 | :-- | :-- | :-- | :-- | :-- | :-- |
 | [Indonesia Convert Cup 2020](ICC/2020) | 13.09.2020 | 08.11.2020 | ::{ flag=ID }:: [dika312](https://osu.ppy.sh/users/741613) | ::{ flag=ID }:: [Galaxi](https://osu.ppy.sh/users/2552435) | ::{ flag=ID }:: [Shurelia](https://osu.ppy.sh/users/3807986) |
 | [Indonesia Convert Cup 2021](ICC/2021) | 16.08.2021 | 11.10.2021 | ::{ flag=ID }:: [Dika312](https://osu.ppy.sh/users/741613) | ::{ flag=ID }:: [Netamaru](https://osu.ppy.sh/users/1830361) | ::{ flag=ID }:: [Madoka Ayukawa](https://osu.ppy.sh/users/1595221) |
+| [Indonesia Convert Cup Open 2022](ICC/2022) | 2022-10-10 | 2022-11-27 | ::{ flag=ID }:: [Netamaru](https://osu.ppy.sh/users/1830361) | ::{ flag=CA }:: [Henrick](https://osu.ppy.sh/users/8840982) | ::{ flag=ID }:: [Chroneko](https://osu.ppy.sh/users/5472877) |
```
Comment (TicClick) [inline wiki/Tournaments/de.md:585]: date format is different

File: wiki/Tournaments/fr.md [hunk 2]
```diff
@@ -582,6 +582,7 @@ Tournois et compétitions non-officiels gérés par la communauté.
 | :-- | :-- | :-- | :-- | :-- | :-- |
 | [Indonesia Convert Cup 2020](ICC/2020) | 13/09/2020 | 08/11/2020 | ::{ flag=ID }:: [dika312](https://osu.ppy.sh/users/741613) | ::{ flag=ID }:: [Galaxi](https://osu.ppy.sh/users/2552435) | ::{ flag=ID }:: [Shurelia](https://osu.ppy.sh/users/3807986) |
 | [Indonesia Convert Cup 2021](ICC/2021) | 16/08/2021 | 11/10/2021 | ::{ flag=ID }:: [Dika312](https://osu.ppy.sh/users/741613) | ::{ flag=ID }:: [Netamaru](https://osu.ppy.sh/users/1830361) | ::{ flag=ID }:: [Madoka Ayukawa](https://osu.ppy.sh/users/1595221) |
+| [Indonesia Convert Cup Open 2022](ICC/2022) | 2022-10-10 | 2022-11-27 | ::{ flag=ID }:: [Netamaru](https://osu.ppy.sh/users/1830361) | ::{ flag=CA }:: [Henrick](https://osu.ppy.sh/users/8840982) | ::{ flag=ID }:: [Chroneko](https://osu.ppy.sh/users/5472877) |
```
Comment (TicClick) [inline wiki/Tournaments/fr.md:585]: date format is different

**Review by TicClick — CHANGES_REQUESTED** [review]

**Comment by TicClick** [comment]
(other than that I see no issues with the ruleset and the article in general)

File: wiki/Tournaments/ICC/2022/en.md [hunk 3]
```diff
@@ -0,0 +1,507 @@
...
+tags:
+  - ICCO2022
+  - ICCO 2022
+---
+
+# Indonesia Convert Cup Open 2022
+
+![Indonesia Convert Cup Open 2022 logo](img/logo.jpg)
+
+The **Indonesia Convert Cup 2022** (***ICC 2022***) was a double-elimination 1v1 osu!catch tournament hosted by ::{ flag=ID }:: [Urabe Mikoto](https://osu.ppy.sh/users/1272422) in which all the beatmaps were osu! converts. Despite its name, the tournament was open to all players from all ten [ASEAN member states](https://asean.org/about-asean/member-states/) as well as ::{ flag=TL }:: Timor-Leste, ::{ flag=NZ }:: New Zealand, and ::{ flag=AU }:: Australia with 3,000 playcounts or more in the osu!catch game mode to their name. It was regarded as the third instalment of the Indonesia Convert Cup tournament series.
```
Comment (Walavouchey) [inline wiki/Tournaments/ICC/2022/en.md:11]:
```suggestion
The **Indonesia Convert Cup Open 2022** (***ICCO 2022***) was a double-elimination 1v1 osu!catch tournament hosted by ::{ flag=ID }:: [Urabe Mikoto](https://osu.ppy.sh/users/1272422) in which all the beatmaps were osu! converts. Despite its name, the tournament was open to all players from all ten [ASEAN member states](https://asean.org/about-asean/member-states/) as well as ::{ flag=TL }:: Timor-Leste, ::{ flag=NZ }:: New Zealand, and ::{ flag=AU }:: Australia with 3,000 play count or more in the osu!catch game mode to their name. It was regarded as the third instalment of the Indonesia Convert Cup tournament series.
```

File: wiki/Tournaments/ICC/2022/en.md [hunk 4]
```diff
@@ -0,0 +1,507 @@
...
+- [Forum thread](https://osu.ppy.sh/community/forums/topics/1649090)
+- [Discord server](https://discord.gg/7g6TAj2)
+- [Challonge brackets](https://challonge.com/ICCO_2022)
+- [Livestream channel](https://www.twitch.tv/osucatchid)
+
+## Participants
+
+Listed below are the players who managed to qualify into the Round of 32 (along with their seed number and average map rank combined across each Qualifier beatmap) out of 39 registered players in total.
+
+| Seed No. | Player | Avg. Map Rank |
```
Comment (Walavouchey) [inline wiki/Tournaments/ICC/2022/en.md:67]:
```suggestion
| Seed no. | Player | Avg. map rank |
```

File: wiki/Tournaments/ICC/2022/en.md [hunk 5]
```diff
@@ -0,0 +1,507 @@
...
+| 27 | ::{ flag=ID }:: [Kinora](https://osu.ppy.sh/users/20571283) | 26.25 |
+| 28 | ::{ flag=PH }:: [-HyperDream](https://osu.ppy.sh/users/12410693) | 26.25 |
+| 29 | ::{ flag=ID }:: [Fisicia](https://osu.ppy.sh/users/6556054) | 26.63 |
+| 30 | ::{ flag=PH }:: [-Isla-](https://osu.ppy.sh/users/17745759) | 29.13 |
+| 31 | ::{ flag=PH }:: [Chloebe](https://osu.ppy.sh/users/4577865) | 29.25 |
+| 32 | ::{ flag=ID }:: [Cera-](https://osu.ppy.sh/users/1974131) | 29.75 |
+
+## Podium
+
+![](img/podium.jpg "ICCO 2022 podium")
```
Comment (Walavouchey) [inline wiki/Tournaments/ICC/2022/en.md:104]:
```suggestion
![ICCO 2022 podium](img/podium.jpg "ICCO 2022 podium")
```

File: wiki/Tournaments/ICC/2022/en.md [hunk 6]
```diff
@@ -0,0 +1,507 @@
...
+2. To ensure that all incoming registrations are serious and valid, every registered player will be checked in detail by the tournament management.
+3. The list of players who are deemed to be eligible to compete in the tournament will be published by the tournament management after the registration phase has ended.
+4. Testplayers, referees, and mappool selectors may not participate as players in this tournament.
+
+### Round-specific rules
+
+#### Qualifier rules
+
+1. Each player will have to sign up to one of the Qualifier lobbies that have been scheduled and prepared by the tournament management in advance.
+2. In the lobby, all players will have to consecutively play all of the nine Qualifier beatmaps in the order of NM1 -> NM2 -> HD1 -> HD2 -> HR1 -> HR2 -> DT1 -> DT2.
```
Comment (Walavouchey) [inline wiki/Tournaments/ICC/2022/en.md:470]: i count 8 in that list

```suggestion
2. In the lobby, all players will have to consecutively play all of the eight Qualifier beatmaps in the order of NM1 -> NM2 -> HD1 -> HD2 -> HR1 -> HR2 -> DT1 -> DT2.
```

File: wiki/Tournaments/ICC/2022/en.md [hunk 7]
```diff
@@ -0,0 +1,507 @@
...
+3. Players **are not allowed** to ban any beatmaps in the Qualifiers.
+4. Players **are not allowed** to join in (or register for) more than one Qualifier lobby.
```
Comment (Walavouchey) [inline wiki/Tournaments/ICC/2022/en.md:472]:
```suggestion
4. Players **are not allowed** to join (or register for) more than one Qualifier lobby.
```

File: wiki/Tournaments/ICC/2022/en.md [hunk 8]
```diff
@@ -0,0 +1,507 @@
...
+1. Each player must use the `!roll` command once in the multiplayer lobby in order to determine the banning and picking order.
+   - The winner of the `!roll` gets to determine who gets the first pick and the second ban.
+   - The loser of the `!roll` gets the opposite by default.
+   - This rule does not apply in the Qualifier lobbies.
+2. Each player has to ban **one beatmap** from the corresponding mappool. These beatmaps will not be allowed to be picked by any player during the entire match.
+   - Barring the tiebreaker, there are no restrictions as to which maps may and may not be banned in a match.
+   - Banning does not apply in the Qualifier lobbies.
+3. Each player is free to select one warm-up beatmap. Using beatmaps with questionable content is prohibited. 
+   - Warm-ups do not apply in the Qualifier lobbies.
+4. **There will be no dedicated FreeMod picks in the tournament.** However, when playing a map from the HardRock or Double Time pool, Hidden can be used optionally on top of the corresponding mod by any player.
```
Comment (Walavouchey) [inline wiki/Tournaments/ICC/2022/en.md:504]:
```suggestion
4. **There will be no dedicated FreeMod picks in the tournament.** However, when playing a map from the HardRock or DoubleTime pool, Hidden can be used optionally on top of the corresponding mod by any player.
```

File: wiki/Tournaments/de.md (hunk 1 above)
Comment (Walavouchey) [reply wiki/Tournaments/de.md:585]:
```suggestion
| [Indonesia Convert Cup Open 2022](ICC/2022) | 10.10.2022 | 27.11.2022 | ::{ flag=ID }:: [Netamaru](https://osu.ppy.sh/users/1830361) | ::{ flag=CA }:: [Henrick](https://osu.ppy.sh/users/8840982) | ::{ flag=ID }:: [Chroneko](https://osu.ppy.sh/users/5472877) |
```

File: wiki/Tournaments/fr.md (hunk 2 above)
Comment (Walavouchey) [reply wiki/Tournaments/fr.md:585]:
```suggestion
| [Indonesia Convert Cup Open 2022](ICC/2022) | 10/10/2022 | 27/11/2022 | ::{ flag=ID }:: [Netamaru](https://osu.ppy.sh/users/1830361) | ::{ flag=CA }:: [Henrick](https://osu.ppy.sh/users/8840982) | ::{ flag=ID }:: [Chroneko](https://osu.ppy.sh/users/5472877) |
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]
just going to quickly skim this one (seems very good writing-wise)

**Review by TicClick — APPROVED** [review]

File: wiki/Tournaments/fr.md [hunk 9]
```diff
@@ -582,6 +582,7 @@ Tournois et compétitions non-officiels gérés par la communauté.
 | :-- | :-- | :-- | :-- | :-- | :-- |
 | [Indonesia Convert Cup 2020](ICC/2020) | 13/09/2020 | 08/11/2020 | ::{ flag=ID }:: [dika312](https://osu.ppy.sh/users/741613) | ::{ flag=ID }:: [Galaxi](https://osu.ppy.sh/users/2552435) | ::{ flag=ID }:: [Shurelia](https://osu.ppy.sh/users/3807986) |
 | [Indonesia Convert Cup 2021](ICC/2021) | 16/08/2021 | 11/10/2021 | ::{ flag=ID }:: [Dika312](https://osu.ppy.sh/users/741613) | ::{ flag=ID }:: [Netamaru](https://osu.ppy.sh/users/1830361) | ::{ flag=ID }:: [Madoka Ayukawa](https://osu.ppy.sh/users/1595221) |
+| [Indonesia Convert Cup Open 2022](ICC/2022) | 10/10/2022 | 17/11/2022 | ::{ flag=ID }:: [Netamaru](https://osu.ppy.sh/users/1830361) | ::{ flag=CA }:: [Henrick](https://osu.ppy.sh/users/8840982) | ::{ flag=ID }:: [Chroneko](https://osu.ppy.sh/users/5472877) |
```
Comment (Walavouchey) [inline wiki/Tournaments/fr.md:585]:
```suggestion
| [Indonesia Convert Cup Open 2022](ICC/2022) | 10/10/2022 | 27/11/2022 | ::{ flag=ID }:: [Netamaru](https://osu.ppy.sh/users/1830361) | ::{ flag=CA }:: [Henrick](https://osu.ppy.sh/users/8840982) | ::{ flag=ID }:: [Chroneko](https://osu.ppy.sh/users/5472877) |
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**